src/SearchResults.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import Moment from "moment";
+
+export default function SearchResults(props) {
+  // let bookings = [
+  //   {
+  //     id: 1,
+  //     title: "Mr",
+  //     firstName: "John",
+  //     surname: "Doe",
+  //     email: "johndoe@doe.com",
+  //     roomId: 2,
+  //     checkInDate: "2017-11-21",
+  //     checkOutDate: "2017-11-23"
+  //   },
+  //   {
+  //     id: 2,
+  //     title: "Doctor",
+  //     firstName: "Sadia",
+  //     surname: "Begum",
+  //     email: "begum_sadia@sadia.org",
+  //     roomId: 1,
+  //     checkInDate: "2018-02-15",
+  //     checkOutDate: "2018-02-28"
+  //   }
+  // ];
+  let bookings = props.results;
+  return (
+    <div>
+      <table class="table">
+        <thead>
+          <tr>
+            <th scope="col">Title</th>
+            <th scope="col">First name</th>
+            <th scope="col">Surname</th>
+            <th scope="col">Email</th>
+            <th scope="col">Room ID</th>
+            <th scope="col">Check-in Date</th>
+            <th scope="col">Check-out Date</th>
+            <th scope="col">Number of nights</th>
+          </tr>
```

Comment:
you could have a think about a dynamic way of grabbing the column headers (see if the object keys may help with this) so you don't have to type them out/update them if the data you are receiving changes